Dockerfile.delft

```diff
@@ -60,33 +61,25 @@ ENV LANG C.UTF-8
 RUN apt-get update && \
     apt-get -y --no-install-recommends install apt-utils build-essential gcc libxml2 unzip curl \
     openjdk-8-jre-headless ca-certificates-java \
-#    git \
     musl gfortran \
     python3 python3-pip python3-setuptools python3-dev \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /opt
-
-COPY --from=builder /opt/grobid-source/grobid-core/build/libs/grobid-core-*-onejar.jar ./grobid/grobid-core-onejar.jar
-COPY --from=builder /opt/grobid-source/grobid-service/build/distributions/grobid-service-*.zip ./grobid-service.zip
-COPY --from=builder /opt/grobid-source/grobid-home/build/distributions/grobid-home-*.zip ./grobid-home.zip
+WORKDIR /opt/grobid
 
-RUN unzip -o ./grobid-service.zip -d ./grobid && \
-    mv ./grobid/grobid-service-* ./grobid/grobid-service
+COPY --from=builder /opt/grobid .
 
-RUN unzip ./grobid-home.zip -d ./grobid && \
-    mkdir -p /opt/grobid/grobid-home/tmp
-RUN rm *.zip
 RUN rm -rf /opt/grobid/grobid-home/pdf2xml/lin-32
 RUN rm -rf /opt/grobid/grobid-home/pdf2xml/mac-64
 RUN rm -rf /opt/grobid/grobid-home/pdf2xml/win-*
 RUN rm -rf /opt/grobid/grobid-home/lib/lin-32
 RUN rm -rf /opt/grobid/grobid-home/lib/win-*
 
-# below to allow logs to be written in the container
+# below to allow logs to be written in the container (not advised for production!)
 # RUN mkdir -p logs
```

Comment:
I've found that even if this directory is not present, log files will still be written to the container. To not write the logs `/opt/grobid/grobid-service/config/config.yaml` has to be updated so that the only appender is `CONSOLE`.  This was discussed in #703, but that change was not made.